### ISSUE TITLE
ODROID-C2: Docker enabled kernel package (provided by HK) is available  

### ISSUE THREAD

**Issue by umiddelb** [hypriot/image-builder-odroid-c2#7]
here: [http://deb.odroid.in/c2/pool/main/l/linux-image-c2/linux-image-c2_35-1_arm64.deb](http://deb.odroid.in/c2/pool/main/l/linux-image-c2/linux-image-c2_35-1_arm64.deb)

**Comment by AstroFrank** [comment]
404 Not Found - No file.

**Comment by umiddelb** [comment]
Somehow I managed to mangle the link (now corrected). But anyway, you should reference the meta package (linux-image-c2) in favour of this one, see [here](https://github.com/umiddelb/z2d/blob/master/common/common-functions.sh#L110-L115).

**Comment by StefanScherer** [comment]
Thanks for the update. It seems that we already have it [here](https://github.com/hypriot/image-builder-odroid-c2/blob/master/builder/chroot-script.sh#L10-L12) and [here](https://github.com/hypriot/image-builder-odroid-c2/blob/master/builder/chroot-script.sh#L50).

I just start a rebuild with a new version v0.0.4 to build a new SD image to update the kernel package.

**Comment by StefanScherer** [comment]
Just a `TAG=v0.0.4 make tag` to trigger a new TravisCI build. Stay tuned and test the release once it is uploaded to the GitHub releases.

**Comment by StefanScherer** [comment]
A new SD card image is here https://github.com/hypriot/image-builder-odroid-c2/releases/tag/v0.0.4

**Comment by umiddelb** [comment]
This is supposed to work, but still without docker, since the Aarch64 repo needs to be added first, see [here](https://github.com/hypriot/image-builder-odroid-c2/commit/1b324b3579d43dda8a6f0cc3dbb82f08301ac90f).

**Comment by marcellodesales** [comment]
@umiddelb @StefanScherer Is there any update on this? Trying to get Docker on `ODroid-C2` and having a hard time using Ubuntu 16.04... (Definitely missing HypriotOS)...

**Comment by umiddelb** [comment]
Hm, did you try

```
sudo apt -y install lxc aufs-tools cgroup-lite apparmor docker.io
```

?

**Comment by umiddelb** [comment]
If you don't like Ubuntu you may try [Armbian](http://www.armbian.com/odroid-c2/) or the [Debian community image](http://forum.odroid.com/viewtopic.php?f=138&t=19403). As they are using the same kernel, Docker should run fine there as well. Just install Docker via:

```
sudo apt-get -y install lxc aufs-tools cgroupfs-mount cgroup-bin apparmor
sudo apt-get -y -t jessie-backports install docker.io
```

**Comment by marcellodesales** [comment]
@umiddelb Yes I like Ubuntu :) So, I will try to flash 0.0.4 and try to install with `sudo apt -y install lxc aufs-tools cgroup-lite apparmor docker.io` you suggested...

Does the other stuff work?
- Wifi
- Bluetooth

?

**Comment by umiddelb** [comment]
You'd better take the [official ODROID C2 Ubuntu Image](http://odroid.com/dokuwiki/doku.php?id=en:c2_ubuntu_release_note_v1.2) then.

**Comment by marcellodesales** [comment]
@umiddelb I've been using Ubuntu Mate, but I'd rather use HypriotOS :)

**Comment by umiddelb** [comment]
Ok, but there is no Aarch64 repository for Hypriot's Docker and friends at this time. 

If you don't need a GUI you may install the minimal Ubuntu / Debian image from ODROID for the C2, install Docker from the repository, activate zram and compile Docker 1.12 from source on the C2.
